deno/lib/__tests__/nullable.test.ts

```diff
@@ -5,7 +5,7 @@ const test = Deno.test;
 import * as z from "../index.ts";
 
 function checkErrors(a: z.ZodTypeAny, bad: any) {
-  let expected;
+  let expected: any;
```

Comment:
This should be typed to whatever `formErrors` produces. `bad: any` should be `bad: unknown` probably.